packs/github/sensors/github_repository_sensor.py

```diff
@@ -34,9 +34,14 @@ def __init__(self, sensor_service, config=None, poll_interval=None):
         self._last_event_ids = {}
 
     def setup(self):
-        self._client = Github(self._config['token'] or None)
+        # Empty string '' is not ok but None is fine. (Sigh)
+        self._client = Github(self._config.get('token', None) or None)
 
-        for repository_dict in self._config['repository_sensor']['repositories']:
+        repository_sensor = self._config.get('repository_sensor', None)
+        if repository_sensor is None:
+            raise ValueError('"repository_sensor" config value is required.')
+
+        for repository_dict in repository_sensor.get('repositories', []):
```

Comment:
Repository sensor without at least one repository specified doesn't really make sense.

In any case, LGTM.